File: Modul 2 Sem 1/Phython/PRAK204-2410817110005-MuhammadRizqanKarim.py
#test case 1
radius1 = int(input(" "))
tinggi1 = int(input(" "))

volume1 = 22/7 * radius1 ** 2 * tinggi1
luas1 = 2 * 22/7 * radius1 * (radius1 + tinggi1)
keliling1 = 2 * 22/7 * radius1

print(" ")
print(f"Volume = {volume1:.2f}")
print(f"Luas = {luas1:.2f}")
print(f"Keliling = {keliling1:.2f}")

#test case 2
radius, tinggi = map(int, input(" "). split())

volume = 22/7 * radius ** 2 * tinggi
luas = 2 * 22/7 * radius * (radius + tinggi)
keliling = 2 * 22/7 * radius

print(" ")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul 2 Sem 1/C/PRAK204-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main()
{
    //test case 1
    float radius1, tinggi1, keliling1, luas1, volume1;

    scanf("%f", &radius1);
    scanf("%f", &tinggi1);

    volume1 = 22.0/7.0 * radius1 * radius1 * tinggi1;
    luas1 = (2 * 22.0/7.0 * radius1 * (radius1 + tinggi1));
    keliling1 = 2 * 22.0/7.0 * radius1;

    printf("\nVolume = %.2f\n", volume1);
    printf("Luas = %.2f\n", luas1);
    printf("Keliling = %.2f\n", keliling1);

    //test case 2
    float radius, tinggi, keliling, luas, volume;

    scanf("%f %f", &radius, &tinggi);

    volume = 22.0/7.0 * radius * radius * tinggi;
    luas = (2 * 22.0/7.0 * radius * (radius + tinggi));
    keliling = 2 * 22.0/7.0 * radius;

    printf("\nVolume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}
